docs/install.md

```diff
@@ -1,52 +1,113 @@
-[ceph-CSI](https://github.com/ceph/ceph-csi/) is one such storage provider
+[Ceph-CSI](https://github.com/ceph/ceph-csi/) is one such storage provider
 that supports the required extensions.
 
-The managed cluster orchestration is provided by Ramen using the
-[VolumeReplicationGroup(VRG) CRD](docs/vrg-crd.md).
-
-**NOTE:** Ramen on OCM managed clusters and its orchestration of VRG CRD can
-be accomplished independent of an OCM setup, and only requires a native
-kubernetes cluster to manage VRG resources.
+[Ramen cluster](#ramen-cluster) component hence should be deployed to OCM
+managed clusters that support VolumeReplication extensions.
 
 ### S3 store
 
-Ramen preserves metadata related to VolumeReplication resources in an S3
-compatible object store. An S3 store endpoint is hence required as part of the
-setup.
+Ramen preserves cluster data related to PVC resources in an S3 compatible
+object store. An S3 store endpoint is hence required as part of the setup.
+
+**NOTE**: Ramen specifically stores PV cluster data for a replicated PVC, to
+restore the same across peer cluster prior to deploying the PVCs of the
+workload, this ensures proper binding of the PVC resources to the replicated
+storage end points.
+
+### Operator lifecycle manager (OLM)
+
+Ramen components are provided as [OLM](https://olm.operatorframework.io/docs/getting-started/)
+catalog sources in the [Ramen catalog](https://quay.io/repository/ramendr/ramen-operator-catalog?tab=info).
+
+All clusters that require Ramen hub or cluster components installed, require
+OLM installed on the same.
+
+### Kubernetes versions
+
+Kubernetes versions supported are [1.20](https://kubernetes.io/releases/)
+or higher.
+
+### Tool versions
+
+Installation and deployment require the following tools at specified versions
+(or higher):
+
+- kubectl > v1.21
+    - kubectl version can be verfied using
+
+      ```bash
+      kubectl version
+      ```
+
+## Ramen hub
+
+`ramen-hub-operator` is the controller for managing the life cycle of user
+created [DRPlacementControl (DRPC)](drpc-crd.md) Ramen API resources and
+administrator created [DRPolicy](drpolicy-crd.md) Ramen API resources, and is
+installed on the **OCM hub cluster**.
+
+### Install ramen-hub-operator
+
+To install `ramen-hub-operator` configure [kubectl](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context)
+to use the desired OCM hub cluster and execute:
+
+```bash
+kubectl apply -k github.com/RamenDR/ramen/config/olm-install/hub/?ref=main
+```
+
+**NOTE**: By default `ramen-hub-operator` creates a deployment for its
+controller in the `ramen-system` namespace. To verify check the health of the
+deployment:
+
+```bash
+kubectl get deployments -n ramen-system ramen-hub-operator
+```
+
+## Ramen cluster
+
+`ramen-dr-cluster-operator` is the controller for managing the life cycle of
+[VolumeReplicationGroup](vrg-crd.md) Ramen API resources and is installed on
+the **OCM managed clusters**.
```

Comment:
It may help to state that ramen-hub-operator auto-creates the VRG resource and that users should neither create nor delete them when using Ramen along with OCM.

Reply:
This should go to the usage doc, that will show by example how a workload is protected etc.